docs/sdk-and-tools/indices/accountshistory.md

```diff
@@ -0,0 +1,38 @@
+---
+id: es-index-accountshistory
+title: Elasticsearch index: accountshistory
+---
+
+The `_id` field of this index is composed in this way: `{bech32address}_{timestamp}` (example: `erd.._1234`).
+
+| Field     | Description                                                                            |
+|-----------|----------------------------------------------------------------------------------------|
+| address   | The address field holds the address bytes in a bech32 encoding.                        |
+| balance   | The balance field holds the amount of EGLD the address has.                            |
```

Comment:
apply the same suggestions for the `address` and `balance` fields' description

Reply:
done